src/averageAges.js

```diff
@@ -37,7 +51,28 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(x => x['sex'] === 'f');
```

Comment:
Same here, use ternary operator, find correct array of people and call reduce for it
```suggestion
  const women = withChildren
    ? people.filter(...)
    : people.filter(x => x['sex'] === 'f');
```